src/Core/SystemAbstractions/IProcess.cs

```diff
@@ -21,19 +21,86 @@
 using System;
 using System.Diagnostics;
 
+/// <summary>
+/// Wrappers around System.Diagnostics.Process for testing
+/// </summary>
```

Comment:
More GitHub confusion.
Moved from `CFamily` and namespace changed.
The interface and class are now public so they are visible to the CFamily assembly.